### ISSUE TITLE
API Issue: ER_PARSE_ERROR: You have an error in your SQL syntax;

### ISSUE THREAD

**Issue by radleta** [andris9/mailtrain#47]
I started integrating the API and it seems to work the first time. If a user signs up again with the same email, I'm receiving the below issue. It might be the version of MySQL 5.7 verses MySQL 5.5 what you are targeting or not. It looked suspicious.

Does this work successfully for you?

Input below.

```Javascript
var addSubscriber = {
    EMAIL: input.email,
    // FIRST_NAME: null,
    // LAST_NAME: null,
    // TIMEZONE: null,
    FORCE_SUBSCRIBE: 'yes',
    REQUIRE_CONFIRMATION: 'no',
  };

  var mailtrainRequestOptions = {
    uri: 'http://***/api/subscribe/***?access_token=***',
    method: 'POST',
    form: addSubscriber,
    json: true,
  };
```

Response below.

```Json
{"error":"ER_PARSE_ERROR: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'WHERE id=403824 LIMIT 1' at line 1","data":[]}
```

**Comment by andris9** [comment]
This is a bug. Fixed and tagged the code as v1.8.1. Update your code and the issue should go away

**Comment by radleta** [comment]
Thanks! I'll update and review.
